src/app/epics/get-items.epic.ts

```diff
@@ -23,19 +24,21 @@ export class GetItemsEpic {
         private searchHistoryActions: SearchHistoryActions,
     ) {}
 
-    getDefaultItems = (action$): Observable<AnyAction> => {
+    getDefaultItems = (action$: ActionsObservable<AnyAction>): Observable<AnyAction> => {
         return action$.ofType(ItemsActions.GET_DEFAULT_ITEMS)
             .do( (action: AnyAction) => console.log('Epic sees action: ', action.type) )
             .mergeMap( () => this.appService.getTrendingItems() )
             .map((items:IItemsRef) => this.itemsActions.recievedDefaultItems(items, Date.now()) );
```

Comment:
Worth mentioning there's a typo here in your method `recievedDefaultItems` => `receivedDefaultItems` :)